### ISSUE TITLE
Questions about test metrics.py . I hope to get your help

### ISSUE THREAD

**Issue by kidwhh** [fvmassoli/cross-resolution-face-recognition#9]
I have read your paper Cross-Resolution Learning for Face Recognition. I run the same test on QMUL-Survface and Tinyface. Test the base models like Sphereface and Vaggface, but the results are much worse, I followed the steps provided by the data set, used the model to generate the feature file, and then ran his MATLAB code. 
	I hope you can provide me with your code for using the model to generate the feature file and then generate the distance matrix. These are the generated files for distance_matrix, probe_features_norm, and so on in your code metrics.py ,let me to call metrics.py directly to reproduce the result.I hope you can help me. 
	Thank you very much.

**Comment by fvmassoli** [comment]
Results from Sphereface and other models are taken from the cited papers.

**Comment by kidwhh** [comment]
@fvmassoli
Thank you very much for your reply
I know these results from are taken from the cited papers.😊
You may have misunderstood me. What I mean  is that I hope you to provide the code that load the Survface test photo by your model，then generates the data and variable required for the metrics.py file. Let me can directly run the metrics. py file to reproduce your model test results.

**Comment by supersyz** [comment]
Hi,I trained the net and get 90% accuracy on 24*24 pixcels  validation LR images,then I test on Tinyface I only get 10% r1 precision, Would you mind give your trained checkpoint to me ?  And when applied in TinyFace, which layer should I choose, The last 8000+ dimensions of features or the  Second to last layer with output of 2000+ dimensions of features?Is the problem that I should tune the model more?

**Comment by fvmassoli** [comment]
I think your results are promising. Just be sure that your model did not overfit.
Concerning the results on Tinyface you should take the features after the avgpool layer, i.e., 20148-dim features.
I did not tune the model on the Tinyface, but I remember that I had to train the model a lot before getting good results on other datasets. Moreover, since not all the local minima are equivalently good (unfortunately) I had to trained several models. 
I can't give you the best paper's exact checkpoint since the research was financially supported by industries that now retain the model's ownership. However, if you send to me an official letter from your academic institution, signed by two full-time professors, in which you declare precisely what your intents are and for what reason you are asking the model, I can share a model's checkpoint pretty close to the best one of the paper.

**Comment by supersyz** [comment]
Thanks  a lot!

**Comment by Zmmccc** [comment]
> Hi,I trained the net and get 90% accuracy on 24*24 pixcels validation LR images,then I test on Tinyface I only get 10% r1 precision, Would you mind give your trained checkpoint to me ? And when applied in TinyFace, which layer should I choose, The last 8000+ dimensions of features or the Second to last layer with output of 2000+ dimensions of features?Is the problem that I should tune the model more?

Hello, excuse me, how can I test on tinyface without a test file for this code? Hope to get your reply.
